webrender/src/renderer.rs

```diff
@@ -141,6 +141,8 @@ impl VertexDataTexture {
 }
 
 const TRANSFORM_FEATURE: &'static [&'static str] = &["TRANSFORM"];
+const YUV_TRANSFORM_FEATURE: &'static [&'static str] = &["TRANSFORM", "YUV_IMAGE"];
```

Comment:
I changed how features work in this PR - https://github.com/servo/webrender/pull/527 - once that lands it should be possible to simplify this a bit (take a look at how the text subpx shader is created).